.cirrus.yml

```diff
@@ -241,7 +247,7 @@ success_task:
     depends_on:
         - validate
         - doccheck
-        - osx
+#        - osx
```

Comment:
If you clobber the commented-out task, make sure to remove this line also.

Reply:
I was on the fence for the clobber, I'll go do so and will move the yaml change to a new commit.